go-controller/pkg/ovn/master.go

```diff
@@ -1374,6 +1374,9 @@ func (oc *Controller) addUpdateNodeEvent(node *kapi.Node, nSyncs *nodeSyncs) err
 		klog.V(5).Infof("When adding node %s, found %d pods to add to retryPods", node.Name, len(pods.Items))
 		for _, pod := range pods.Items {
 			pod := pod
+			if util.PodCompleted(&pod) {
+				continue
```

Comment:
unless we were retrying a deletion from previous cluster? or actually I guess we don't have to worry about this since reboots and retries won't go well together? retry handler isn't persistent is it?

Reply:
Doesn't look like we clean up NAT entries, but it shouldn't break anything. Whenever a new pod is added, egress gw/ip is used it should handle the NATs. Something to note for the future though in case we want to clean it up.